recipes/modern-cpp-kafka/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class ModernCppKafkaConan(ConanFile):
+    name = "modern-cpp-kafka"
+    description = "A C++ API for Kafka clients (i.e. KafkaProducer, KafkaConsumer, AdminClient)"
+    license = "Apache License 2.0"
+    topics = ("conan", "modern-cpp-kafka", "kafka", "librdkafka", "kafkaproducer", "kafkaconsumer")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/morganstanley/modern-cpp-kafka"
+    exports_sources = "include/*"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+       tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst="include", src=os.path.join(self._source_subfolder, "include"))
+
+    def package_id(self):
+        self.info.header_only()
+
```

Comment:
Package info is missing. Conan need to generate `RdKafka::rdkafka` according.

Please, read https://docs.conan.io/en/latest/creating_packages/package_information.html#using-components and https://docs.conan.io/en/latest/migrating_to_2.0/properties.html

Also, pthread is a mandatory dependency https://github.com/morganstanley/modern-cpp-kafka/blob/main/CMakeLists.txt#L50

Reply:
Hi, @uilianries I just updated the commit. Please help review again. Thank you a lot! 😀